spec/granite_orm/fields/timestamps_spec.cr

```diff
@@ -0,0 +1,67 @@
+require "../../spec_helper"
+
+# TODO sqlite support for timestamps
+{% for adapter in ["pg", "mysql"] %}
+  {%
+    parent_constant = "Parent#{adapter.camelcase.id}".id
+
+    # TODO mysql timestamp support should work better
+    if adapter == "pg"
+      time_kind_on_read = "Time::Kind::Utc".id
+    elsif adapter == "mysql"
+      time_kind_on_read = "Time::Kind::Unspecified".id
```

Comment:
@drujensen, @eliasjpr since granite currently has some functionality out of parity across adapters, this spec is a little bespoke and needs to be more explicit than the rest. I'm assuming that the adapters will be brought into parity at some point and this can be cleaned up. #29 and #97 for example